course/frontend/polyfills/scripts/main.js

```diff
@@ -8,6 +8,6 @@ a.forEach(function(v) {
 
```

Comment:
HEY! this is not part of arrays challenge. PLEASE, pay attention when submitting code in a branch. it should ONLY have changes corresponding to a specific challenge, not messing with other stuff.